Documentation/Building the Extension.md

```diff
@@ -13,3 +13,12 @@ Prerequisite steps:
   * **(optional)** Set an environment variable `CPPTOOLS_DEV=1`.
     * This enables the local developer workflow when testing the debugger, copying dependencies from the **node_modules** folder. Testing the language server does not require this step.
   * Open the **Extension** folder in Visual Studio Code and press F5. This will launch a VS Code Extension Host window and activate the TypeScript debugger. You can set breakpoints on the extension source code and debug your scenario.
+      * If, after pressing F5, you see the following error in the `[Extension Development Host]` window,
+      > Unable to start the C/C++ language server. IntelliSense features will be disabled. Error: Missing binary at .../vscode-cpptools/Extension/bin/cpptools
+      * ... Then, you can follow the instructions in this [comment in a discussion about building this extension locally](https://github.com/microsoft/vscode-cpptools/discussions/8745#discussioncomment-2091563).
+        > get the 1.8.2 binaries from installing the extension and then copying the binaries
+        1. To do this, install this extension from the Visual Studio Marketplace and find its location on your device. It might be in a directory like `\\wsl$\Ubuntu\home\hamir\.vscode-server\extensions\ms-vscode.cpptools-1.8.2`, for example.
+        2. Next, go to the `bin/` directory of the aforementioned directory, and drag-and-drop, or copy-and-paste, `cpptools` and `cpptools-srv` from `...\extensions\ms-vscode.cpptools-1.8.2\bin\` to this repository's `Extension\bin\` directory on your local device, so that `.../vscode-cpptools/Extension/bin/cpptools` and `.../vscode-cpptools/Extension/bin/cpptools-srv` both exist in your workspace.
```

Comment:
Can you replace 1.8.2 with something like `<package.version.number>`?

Reply:
Sure! [3d509a2](https://github.com/microsoft/vscode-cpptools/pull/8788/commits/3d509a2d18340583589cdec10266f9971edd4e0f) should address this.